src/WebhookSignature.php

```diff
@@ -19,7 +19,7 @@ public function __construct($secret)
      */
     public function validate($signature, $payload)
     {
-        $payloadSignature = 'sha1='.hash_hmac('sha1', $payload, $this->secret);
+        $payloadSignature = 'sha1='.hash_hmac('sha1', $payload ?? '', $this->secret);
```

Comment:
Can you also update the method signature to indicate that `null` is now a valid value for `$payload`?